exporters/otlp/otlplog/otlploghttp/client_test.go

```diff
@@ -779,3 +779,39 @@ func TestConfig(t *testing.T) {
 		assert.Equal(t, []string{headerValueSetInProxy}, got[headerKeySetInProxy])
 	})
 }
+
+// borrows from TestConfig.
+func TestNonRetryable(t *testing.T) {
```

Comment:
Could this be a subtest in `TestConfig` instead, like the other test cases?

Reply:
I didn't consider it a configuration test, but I can put it there if that works better for you.

Reply:
Retryable errors test is in there.
Maybe that test could be refactored. But we should do that in another PR.